test_harness/src/lib.rs

```diff
@@ -765,14 +995,45 @@ mod tests {
                 balances: Balances::<CheckedState>::new(),
             };
             let actual_accounting = leader_sentry
-                .get_accounting(&token_chain.clone().with_channel(CAMPAIGN_1.channel))
+                .get_accounting(&token_chain_1337.clone().with_channel(CAMPAIGN_1.channel))
                 .await
                 .expect("Should get Channel Accounting");
 
             assert_eq!(expected_accounting, actual_accounting);
+            info!(setup.logger, "Channel 1 {:?} has empty Accounting because of no events have been submitted to any Campaign", CAMPAIGN_1.channel.id());
         }
 
-        // Add new events to sentry
+        // All Channels should have a heartbeat message now
+        // Channel 1
+        // Channel 2
+        // Channel 3
+        {
+            // Channel 1
+            let _channel_1_heartbeat = leader_sentry
+                .get_our_latest_msg(CAMPAIGN_1.channel.id(), &["Heartbeat"])
+                .await
+                .expect("Should fetch Heartbeat from Leader")
+                .map(|message| Heartbeat::try_from(message).expect("Should be Heartbeat"))
+                .expect("Should have a Heartbeat in Leader for the Campaign 1 channel");
+
+            // Channel 2
+            let _channel_2_heartbeat = leader_sentry
+                .get_our_latest_msg(CAMPAIGN_2.channel.id(), &["Heartbeat"])
+                .await
+                .expect("Should fetch Heartbeat from Leader")
+                .map(|message| Heartbeat::try_from(message).expect("Should be Heartbeat"))
+                .expect("Should have a Heartbeat in Leader for the Campaign 2 channel");
+
+            // Channel 3
+            let _channel_2_heartbeat = leader_sentry
```

Comment:
```suggestion
            let _channel_3_heartbeat = leader_sentry
```

The unused variable is not named correctly.